src/editors/containers/ProblemEditor/index.jsx

```diff
@@ -2,22 +2,30 @@ import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Spinner } from '@edx/paragon';
-import { injectIntl } from '@edx/frontend-platform/i18n';
+import { injectIntl, FormattedMessage } from '@edx/frontend-platform/i18n';
 import SelectTypeModal from './components/SelectTypeModal';
 import EditProblemView from './components/EditProblemView';
 import { selectors, thunkActions } from '../../data/redux';
 import { RequestKeys } from '../../data/constants/requests';
+import messages from './messages';
 
 export const ProblemEditor = ({
   onClose,
   // Redux
   problemType,
   blockFinished,
+  blockFailed,
   studioViewFinished,
   blockValue,
   initializeProblemEditor,
   assetsFinished,
 }) => {
+  React.useEffect(() => {
```

Comment:
`initializeProblemEditor` seems to be a line that is not covered by tests - probably this is what codecov is complaining about? Up to now we usually have useEffects and such in the hooks file and then tested by mocking out effects and such. That could be a solution here if you don't want to use new tooling.

I'm just trying to suggest anything that could fix the code coverage problem, but go ahead and fix that in any way you see fit, I don't have any preference.
- Wile the place where this useEffect logic is right now is not consistent with the rest of the application, that is not due to you and not something we need to fix now.

Reply:
isn't "test `initializeProblemEditor`" the whole point of the (currently excluded) first bullet in the ticket?